app/partials/events.html

```diff
@@ -0,0 +1,24 @@
+<div cg-busy="{promise: loadingPromise, templateUrl:'partials/loadingtemplate.html'}">
+<span ng-if="events">
+    <section id="events">
+        <div ng-if="events.length == 1">
+            <h2>{{events[0].attributes.title}}</h2>
+            <p>{{events[0].attributes.description}}</p>
+            <a role="button" href="/checkin/{{events[0].id}}" class="button flatButton" ui-sref="checkin({id: events[0].id })">Check in to this Event</a>
+        </div>
+        <table ng-if="events.length > 1" ng-repeat="event in events">
```

Comment:
I don't think this is right — this is repeating the *table*. You want the `ng-repeat` one line below, on the `tr`.